Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:16
+
+WORKDIR /usr/src/app
+
+COPY package.json ./
+COPY yarn.lock ./
```

Comment:
We don't need this ig?

Reply:
The `yarn.lock` file? I think it is important to have the `yarn.lock` file.

Reply:
We don't need it actually because on running yarn install, it'll automatically look for a lock file and if it doesn't find a lockfile then it'll create one.

But no issues, it won't cause any problem if we have it 💪🏻